workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -111,6 +115,16 @@ public RedirectedProcess AddErrorProcessing(Action<string> errorProcessor)
             return this;
         }
 
+        /// <summary>
+        ///     Adds accumulated output processing
+        /// </summary>
+        /// <param name="errorProcessor">Processing action for error output.</param>
+        public RedirectedProcess AddAccumulatedOutputProcessing(Action<string> errorProcessor)
```

Comment:
`errorProcessor` seems like the wrong name :)

Reply:
hmmm, I recall making those changes. Probably lost them during some nasty rebase :(